web/store/message.ts

```diff
@@ -112,14 +113,14 @@ export const msgStore = createStore<MsgState>('messages', {
     async *confirmSwipe({ retries }, msgId: string, position: number, onSuccess?: Function) {
       const replacement = retries[msgId]?.[position]
       if (!retries || !replacement) {
-        return toastStore.error(`Cannot confirm swipe: Swipe state is stale`)
+        return
       }
 
-      const list = retries[msgId]
-      list.splice(position, 1)
-      const next = [replacement].concat(list)
+      // const list = retries[msgId]
```

Comment:
So this code would move the confirmed swipe to the front of the list and the current swipe position would be set to `0`.
Is the idea now that the order never changes?

I figured in a flow that looks like this:

- Bot responds
- Retry, retry, retry, retry (swipes now shows 1/5)
- Swipe to 3/5
- Confirm
- User responds
- Bot responds
- User deletes bot response + user response
- Swipe position is back to `0` which is wrong, it should be `2` (which represents 3/5)

Reply:
>Is the idea now that the order never changes?

yea, instead of confirming with a checkmark the user confirms when they respond to the response they picked by swiping

>User deletes bot response + user response
Swipe position is back to `0` which is wrong, it should be `2` (which represents 3/5)

with latest commit when a new response is generated the swipe is set to the latest one. when a user confirms by responding to their selected response, if they delete the new response to go back to the old message swipe is set the the last swipe in that messages retires